content/docs-community/concepts/contributing-to-docs/terms/docs-repository/docs-repository.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Docs Repository'
+Description: 'The docs repository is where contributors can view, edit, and propose new content to be published to the Codecademy Docs site.'
+Subjects:
+  - 'Code Foundations'
+Tags:
+  - 'Documentation'
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'introduction-to-open-source'
+  - 'paths/code-foundations'
+---
+
+The content-base where Codecademy Docs is housed is the [**`docs` repository**](https://github.com/Codecademy/docs) (or "repo") on [GitHub](https://www.codecademy.com/resources/docs/general/github). Contributors can view, edit, and propose new content for the repo. After the changes are merged into the repo's `main` branch, they will go live on the [Docs site](https://www.codecademy.com/resources/docs).
+
+## Forking the Repository
+
+When getting started, contributors should create a new fork of the `docs` repo. This should be done prior to changing any existing files or creating new ones in order to avoid merge conflicts.
+
+![A red stencil outlining the "Fork" link for the Codecademy Docs repo](https://raw.githubusercontent.com/Codecademy/docs/main/media/indicate_fork_link.png)
+
+The forked repo is a remote repo that exists on GitHub. To make changes on a local computer, a copy of the remote `docs` repo (local repo) will need to be made using the following command:
```

Comment:
To make changes on a local computer, a copy of the remote `docs` repo (local repo) ~will need to~ must be be made using the following command